src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -26,14 +33,15 @@ public class DeleteCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
                 + ": Deletes the people identified by the index numbers used in the displayed person list.\n"
-                + "Parameters: INDEXES (must be a positive integer)\n"
+                + "Parameters: INDEXES (must be positive integers not exceeding max integer value)\n"
```

Comment:
Maybe can specify the max integer value so it would be this instead: `must be positive integers not exceeding 2^31 - 1`.
Personally, this change is not necessary as user input usually contain a few digits. Perhaps this note could be written in the user guide instead.